### PR TITLE
Add js-challenges

### PR DESCRIPTION
**Pull request by lgimeneza**

### CODE REVIEW

**Review by MinaZhen — APPROVED** [review]
+1!

File: staff/lgimeneza/js/challenges/scripts/toRandomCase.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+function toRandomCase(text){
```
Comment (agandia9) [inline staff/lgimeneza/js/challenges/scripts/toRandomCase.js:3]: use strict!

File: staff/lgimeneza/js/challenges/scripts/toRandomCase.test.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+// Test case lower case input is equals to output
```
Comment (agandia9) [inline staff/lgimeneza/js/challenges/scripts/toRandomCase.test.js:3]: use strict!

File: staff/lgimeneza/js/challenges/scripts/toRomanNumeral.test.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// Test case with valid númber
```
Comment (agandia9) [inline staff/lgimeneza/js/challenges/scripts/toRomanNumeral.test.js:3]: use strict!

File: staff/lgimeneza/js/challenges/scripts/toRomanNumeral.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+'use strict';
```
[the remark this answers is not in the document]
Comment (lgimeneza) [reply staff/lgimeneza/js/challenges/scripts/toRomanNumeral.js:1]: 🙄

File: staff/lgimeneza/js/challenges/scripts/toRomanNumeral.test.js (hunk 3 above)
Comment (lgimeneza) [reply staff/lgimeneza/js/challenges/scripts/toRomanNumeral.test.js:3]: done!

File: staff/lgimeneza/js/challenges/scripts/toRandomCase.test.js (hunk 2 above)
Comment (lgimeneza) [reply staff/lgimeneza/js/challenges/scripts/toRandomCase.test.js:3]: done!

File: staff/lgimeneza/js/challenges/scripts/toRandomCase.js (hunk 1 above)
Comment (lgimeneza) [reply staff/lgimeneza/js/challenges/scripts/toRandomCase.js:3]: done!

**Pull request merged**